src/CameraControls.ts

```diff
@@ -1002,7 +1021,8 @@ export class CameraControls extends EventDispatcher {
 				( this._state & ACTION.ROTATE ) === ACTION.ROTATE ||
 				( this._state & ACTION.TOUCH_ROTATE ) === ACTION.TOUCH_ROTATE ||
 				( this._state & ACTION.TOUCH_DOLLY_ROTATE ) === ACTION.TOUCH_DOLLY_ROTATE ||
-				( this._state & ACTION.TOUCH_ZOOM_ROTATE ) === ACTION.TOUCH_ZOOM_ROTATE
+				( this._state & ACTION.TOUCH_ZOOM_ROTATE ) === ACTION.TOUCH_ZOOM_ROTATE ||
+				isPointerLockActive
```

Comment:
How about applying the mouse left button setting for pointerLock?
(If the pointer is locked, it assumes the state as the left button action.

```
			const state = isPointerLockActive ? this.mouseButtons.left : this._state;

			if (
				( state & ACTION.ROTATE ) === ACTION.ROTATE ||
				( state & ACTION.TOUCH_ROTATE ) === ACTION.TOUCH_ROTATE ||
				( state & ACTION.TOUCH_DOLLY_ROTATE ) === ACTION.TOUCH_DOLLY_ROTATE ||
				( state & ACTION.TOUCH_ZOOM_ROTATE ) === ACTION.TOUCH_ZOOM_ROTATE
			) {
```

then do the same for the rest conditions as well.

Reply:
Any downsides to this? I can imagine one scenario, where pointer locked app would like to pan with mouse1 pressed - and they couldn't because mouse1 would be "always on" when pointer is locked.

Reply:
sorry for the lack of information.
I meant if we expose the method, we can set `true` to the flag.

```ts
lockPointer () {
  this._pointerLocked = true;
  // ...snip
  // - call requestPointerLock
  // - enable auto dragging
}
```

then if the pointer lock is called via the method, we will apply the mouse left action.

```diff
- const state = isPointerLockActive ? this.mouseButtons.left : this._state;
+ const state = this._pointerLocked ? this.mouseButtons.left : this._state;
```

If users call `requestPointerLock` in user-land directly, `this._pointerLocked` won't be `true` and they can use original actions.

(Sorry if I was misunderstanding...